scripts/publish.js

```diff
@@ -12,6 +12,17 @@ shell.set('+v');
  * 2. The script is running in the right branch (e.g., release/vxx.y.z)
  */
 
+// Checks that you are running this with Node v7.9.0 and above
+const [version, major, minor, patch] = process.version.match(
+  /^v(\d+)\.?(\d+)\.?(\*|\d+)$/
```

Comment:
Would process.version ever return * for patch?

Reply:
Not sure. I'd rather leave this as-is for now.

Reply:
Ok LGTM